### PR TITLE
Functional Tests : Faceted search module - Uninstall and delete module

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Functional Tests : Faceted search module - Uninstall and delete module
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: & Nightly is :green_circle: : https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/5122831725
| Fixed ticket?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

**Review by matthieu-rolland — COMMENTED** [review]
Everything almost green :+1:   3 autom tests still running...

**Review by boubkerbribri — DISMISSED** [review]

File: tests/UI/pages/FO/category/index.ts [hunk 1]
```diff
@@ -339,6 +344,15 @@ class Category extends FOBasePage {
 
     return null;
   }
+
+  /**
+   * Return
```
Comment (nesrineabdmouleh) [inline tests/UI/pages/FO/category/index.ts]:
```suggestion
   * Is searh filter visible
```

Comment (Progi1984) [reply tests/UI/pages/FO/category/index.ts]: Fixed

**Review by matthieu-rolland — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @Progi1984

**Pull request merged**